unstructured/ingest/v2/cli/cmds/pinecone.py

```diff
@@ -0,0 +1,70 @@
+from dataclasses import dataclass
+
+import click
+
+from unstructured.ingest.v2.cli.base import DestCmd
+from unstructured.ingest.v2.cli.interfaces import CliConfig
+from unstructured.ingest.v2.processes.connectors.pinecone import CONNECTOR_TYPE
+
+
+@dataclass
+class PineconeCliConnectionConfig(CliConfig):
+    @staticmethod
+    def get_cli_options() -> list[click.Option]:
+        options = [
+            click.Option(
+                ["--api-key"],
+                required=True,
+                type=str,
+                help="API key for Pinecone.",
+            ),
+            click.Option(
+                ["--index-name"],
+                required=True,
+                type=str,
+                help="Name of the index to connect to. Example: my-index",
+            ),
+            click.Option(
+                ["--environment"],
+                required=True,
+                type=str,
+                help="Environment to connect to. Example: us-east-1",
+            ),
+        ]
+        return options
+
+
+@dataclass
+class PineconeCliUploaderConfig(CliConfig):
+    @staticmethod
+    def get_cli_options() -> list[click.Option]:
+        options = [
+            click.Option(
+                ["--batch-size"],
+                default=100,
+                type=int,
+                help="Number of records per batch",
+            ),
+            click.Option(
+                ["--num-processes"],
+                default=4,
+                type=int,
+                help="Number of processes to use for uploading",
+            ),
+        ]
+        return options
+
+
+@dataclass
+class PineconeCliUploadStagerConfig(CliConfig):
```

Comment:
Don't need to include this is it's not adding any options

Reply:
[no need to add PineconeCliUploadStagerConfig to pinecone cli cmd as i…](https://github.com/Unstructured-IO/unstructured/pull/3286/commits/5b146ee013ce42851a0a20d91f6d82dc3c77445f)